src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -10,5 +10,7 @@ public class Messages {
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_PROFILE_DISPLAYED_INDEX = "The profile index provided is invalid";
     public static final String MESSAGE_PROFILES_LISTED_OVERVIEW = "%1$d profiles listed!";
-
+    public static final String MESSAGE_PROFILE_LISTED_OVERVIEW = "1 profile listed!";
+    public static final String MESSAGE_MISSING_KEYWORDS = "Provide at least one keyword. \n";
+    public static final String MESSAGE_NO_MATCH = "There are no matching results!";
```

Comment:
Hm, I'm not too sure about this either. My thought process was that since it's in the `commons` package, it should only contain messages that are used across the different classes, rather than just for one specific class. Thoughts, @sheshenk @KSHan29 @ichigh0st?

Reply:
I agree. I think this should be under the `FindProfileCommand` class itself.